.circleci/patch_circleci_environment.js

```diff
@@ -1,11 +1,100 @@
+/**
+ * Patch the CircleCI build environment to fix a known problem with DangerCI
+ * recognizing some pull requests in CircleCI.
+ *
+ * http://danger.systems/js/guides/faq.html#circle-ci-doesnt-run-my-build-consistently
+ *
+ * If you are a contributor to pwa-studio with access to the
+ * magento-research/pwa-studio repo directly, you can push new branches to it
+ * and open pull requests without having to fork to your personal account.
+ *
+ * However, if CircleCI is configured to run builds for every branch, whether
+ * it is a pull request or not, then it will initiate a build for a pushed
+ * branch you have the chance to open a pull request on it. After you have
+ * opened the pull request, CircleCI will continue to reuse the previous
+ * environment it made for the branch before it became a PR. This is a defect
+ * that CircleCI is tracking:
```

Comment:
Missing a word here. Suggestion in bold:

> ...then it will initiate a build for a pushed branch **before** you have the chance to open a pull request on it.

Reply:
Fixed!